include/core/mir/int_wrapper.h

```diff
@@ -70,6 +70,12 @@ inline constexpr bool operator < (IntWrapper<Tag,ValueType> const& lhs, IntWrapp
 {
     return lhs.as_value() < rhs.as_value();
 }
+
+template<typename Tag, typename ValueType>
+inline constexpr bool operator > (IntWrapper<Tag,ValueType> const& lhs, IntWrapper<Tag,ValueType> const& rhs)
+{
+    return lhs.as_value() > rhs.as_value();
+}
```

Comment:
That does fix an omission, but I prefer to fix by modernising the code (#3266)

Reply:
Sure. Much better.